### PR TITLE
feat: add details for TonWallet transaction

### PR DESCRIPTION
**Pull request by Alex-A4**
## Description

copilot:all

## Type of Change

Added details for next transaction types:
- Ordinary
- MultisigOrdinary
- MultisigPending
- MultisigExpired

# Waiting for bridge&nekoton_repo release

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/feature/wallet/widgets/account_transactions_tab/detail/details_custodians.dart [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      spacing: DimensSize.d8,
+      alignment: WrapAlignment.end,
+      runSpacing: DimensSize.d4,
+      children: [
+        if (custodian == initiator)
+          CommonChips(
+            title: LocaleKeys.initiatorWord.tr(),
+            leading: CommonIconWidget.svg(
+              // ignore: no-magic-number
+              size: 14,
```
Comment (nesquikm) [inline lib/feature/wallet/widgets/account_transactions_tab/detail/details_custodians.dart:88]: Whaaat? :)

File: lib/feature/wallet/widgets/account_transactions_tab/detail/details_body.dart [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  final Money value;
+
+  /// Fee of transaction
+  final Money fee;
+
+  /// Status of transaction
+  final TonWalletTransactionStatus status;
+
+  /// Type (ordinary, multisig sumbit/send)
+  // final String type;
```
[the remark this answers is not in the document]
Comment (Alex-A4) [reply lib/feature/wallet/widgets/account_transactions_tab/detail/details_body.dart:45]: -

File: lib/feature/wallet/widgets/account_transactions_tab/detail/details_body.dart [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+                  MoneyWidget(money: fee, style: MoneyWidgetStyle.primary),
+            ),
+            if (comment != null && comment!.isNotEmpty)
+              TonWalletTransactionDetailsItem(
+                title: LocaleKeys.commentWord.tr(),
+                content: comment,
+              ),
+            // TonWalletTransactionDetailsItem(
+            //   title: LocaleKeys.typeWord.tr(),
+            //   content: type,
```
[the remark this answers is not in the document]
Comment (Alex-A4) [reply lib/feature/wallet/widgets/account_transactions_tab/detail/details_body.dart:103]: -

File: lib/feature/wallet/widgets/account_transactions_tab/detail/details_extensions.dart [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+      value,
+      Currencies()[
+          inject<NekotonRepository>().currentTransport.nativeTokenTicker]!,
+    );
+
+extension KnownPayloadX on KnownPayload {
+  /// Returns pair, where 1-st item is title of section,
+  /// 2-nd item list of content specified for payload
+  // ignore: long-method
+  (String, List<TonWalletTransactionDetailsItem>)? toRepresentableData() =>
```
[the remark this answers is not in the document]
Comment (Alex-A4) [reply lib/feature/wallet/widgets/account_transactions_tab/detail/details_extensions.dart:18]: -

**Review by nesquikm — APPROVED** [review]

**Pull request merged**